pasqal_cloud/device/configuration/base_config.py

```diff
@@ -34,6 +34,8 @@ class BaseConfig:
     parameters are added to the configuration dictionary.
     """
 
+    strict_validation: bool = False
+    """Whether to validate an emulator job as though it were for FRESNEL."""
```

Comment:
Is `as though` correct in that context? It reads funny to me but maybe I'm not aware of this formulation in English

Reply:
as though means as if
https://www.collinsdictionary.com/dictionary/english/as-though#:~:text=phrase,he'd%20said%20something%20funny.

Reply:
TIL :)